src/testing.ts

```diff
@@ -76,6 +76,33 @@ const expectedResults = new Map<string, ResultType>([
   ['example_contracts/freeStruct', 'Success'],
   ['example_contracts/function_with_nested_return', 'Success'],
   ['example_contracts/functionArgumentConversions', 'Success'],
+  ['example_contracts/functionInputs/arrayTest/arrayArrayArray', 'Success'],
+  ['example_contracts/functionInputs/arrayTest/arrayArrayDynArray', 'NotSupportedYet'],
+  ['example_contracts/functionInputs/arrayTest/arrayArrayStruct', 'Success'],
+  ['example_contracts/functionInputs/arrayTest/arrayDynArrayArray', 'NotSupportedYet'],
+  ['example_contracts/functionInputs/arrayTest/arrayDynArrayDynArray', 'NotSupportedYet'],
+  ['example_contracts/functionInputs/arrayTest/arrayDynArrayStruct', 'NotSupportedYet'],
+  ['example_contracts/functionInputs/arrayTest/arrayStructArray', 'Success'],
+  ['example_contracts/functionInputs/arrayTest/arrayStructDynArray', 'NotSupportedYet'],
+  ['example_contracts/functionInputs/arrayTest/arrayStructStruct', 'Success'],
+  ['example_contracts/functionInputs/dynArrayTest/dynArrayArrayArray', 'NotSupportedYet'],
+  ['example_contracts/functionInputs/dynArrayTest/dynArrayArrayDynArray', 'NotSupportedYet'],
+  ['example_contracts/functionInputs/dynArrayTest/dynArrayArrayStruct', 'NotSupportedYet'],
+  ['example_contracts/functionInputs/dynArrayTest/dynArrayDynArrayArray', 'NotSupportedYet'],
+  ['example_contracts/functionInputs/dynArrayTest/dynArrayDynArrayDynArray', 'NotSupportedYet'],
+  ['example_contracts/functionInputs/dynArrayTest/dynArrayDynArrayStruct', 'NotSupportedYet'],
+  ['example_contracts/functionInputs/dynArrayTest/dynArrayStructArray', 'Success'],
+  ['example_contracts/functionInputs/dynArrayTest/dynArrayStructDynArray', 'NotSupportedYet'],
+  ['example_contracts/functionInputs/dynArrayTest/dynArrayStructStruct', 'Success'],
+  ['example_contracts/functionInputs/structTest/structArrayArray', 'Success'],
+  ['example_contracts/functionInputs/structTest/structArrayDynArray', 'NotSupportedYet'],
+  ['example_contracts/functionInputs/structTest/structArrayStruct', 'Success'],
+  ['example_contracts/functionInputs/structTest/structDynArrayArray', 'NotSupportedYet'],
+  ['example_contracts/functionInputs/structTest/structDynArrayDynArray', 'NotSupportedYet'],
+  ['example_contracts/functionInputs/structTest/structDynArrayStruct', 'NotSupportedYet'],
+  ['example_contracts/functionInputs/structTest/structStructArray', 'Success'],
+  ['example_contracts/functionInputs/structTest/structStructDynArray', 'NotSupportedYet'],
```

Comment:
As discussed there should be WillNotSupport, at least until we tidy up the wording to be clear that it means Will Not Support until dependencies change, vs NotSupportedYet being used as a "not implemented yet but needs doing"